src/UI/Settings.js

```diff
@@ -1120,6 +1157,7 @@ function Settings(props) {
         <Form onSubmit={handleSubmit}>
           <H3>Change drop shadow</H3>
           <Input
+            defaultValue={theme.drop_shadow}
```

Comment:
You should not rely on the theme, instead rely on the useTheme() object that has access to all the theme attributes like this:
useTheme().primary_color
You have multiple examples in this file to see how it's used.